python/tvm/driver/tvmc/model.py

```diff
@@ -248,11 +284,12 @@ def export_classic_format(
 
     def export_package(
         self,
-        executor_factory: GraphExecutorFactoryModule,
+        executor_factory: Union[GraphExecutorFactoryModule, tvm.runtime.vm.Executable],
```

Comment:
To look prettier we can add `from tvm.runtime.vm import Executable` up top then just use `Executable` here.